packages/web/src/components/tracks-table/TracksTable.module.css

```diff
@@ -43,6 +43,10 @@
   opacity: 0;
 }
 
+.trackActionsContainer {
+  margin-left: 0px;
```

Comment:
nit: Prefer this to go on the actual `Flex` component via prop `ml='0'`

Reply:
oof we don't have a 0 or unit 0 for `ml`

Reply:
oof

Reply:
gross, we should probably extend that type but no worries to add it in this PR